playground/jquery.html

```diff
@@ -30,16 +30,21 @@
     -->
 
     <script type="text/javascript" src="../artifacts/js/dx.all.debug.js" charset="utf-8"></script>
+    <script type="text/javascript" src="../node_modules/axe-core/axe.min.js"></script>
 </head>
 <body>
-    <div id="button"></div>
-    <script>
-        $(function() {
-            $("#button").dxButton({
-                text: 'Click me!',
-                onClick: function() { alert("clicked"); }
+    <div role="main">
+        <h1 style="position: fixed; left: 0; top: 0; clip: rect(1px, 1px, 1px, 1px);">Test header</h1>
```

Comment:
why do we need test header?

Reply:
It is an accessibility requirement and aXe tracks this point.